### ISSUE TITLE
Stripe integration doesn't actually charge the card

### ISSUE THREAD

**Issue by yossizahn** [jwallet/donate#1]
Hi,
Cool project!
It seems that the Stripe support is somewhat unfinished? You only call `createPaymentMethod` which doesn't do the actual charge.

**Comment by jwallet** [comment]
yes I did not have time to go back to it and work to finish it, like I'm not sure if paypal redirects back to the website when the payment is done. that's why the readme here, still does not link to anything when clicking on _Visit the live app_
